products/bluebell/docs/address.md

```diff
@@ -0,0 +1,137 @@
+## `Address` type format
+
+The `Address` data structure in Scilla represents the 20-byte Ethereum-like
```

Comment:
if you are going to discuss addresses here, makes sense to note that they are NOT generated in the same way as eth, as the hashing algo is different

Reply:
Ah yes, that is right. I actually forgot about that